app/controllers/shipments_controller.rb

```diff
@@ -2,4 +2,15 @@ class ShipmentsController < ApplicationController
   def index
     @shipments = Shipment.all
   end
+
+  def show
+    @shipment = Shipment.find(params[:id])
+  end
+
+  private
+
+  def company_params
```

Comment:
I guess you want to set a `before_action` for setting `@company`?

Reply:
Yes that is what I was trying to do but I actually changed it in task 3 as I'm not sure if it was actually necessary. Could you please advise if this is needed?